.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -0,0 +1,72 @@
+---
+name: Bug report
+about: Report a problem with XCSoar to help us resolve it
+---
+
+<!--
+
+Welcome! - We kindly ask that you:
+
+  1. Fill out the issue template below - not doing so needs a good reason.
+  2. Use the forum if you have a question rather than a bug or feature request.
+
+The forum is at: https://forum.xcsoar.org
+
+Thanks for understanding, and for contributing to the project!
+
+-->
+
+
+XCSoar version
+--------------
+
+<!--
+Which XCSoar version are you using?
+-->
+
+
+Expected behavior
+-----------------
+
+<!--
+Please describe what you expected to see / should happen.
+-->
+
+
+Actual behavior
+---------------
+
+<!--
+Please describe the exact symptoms that you were seeing instead.
+-->
+
+
+Steps to reproduce the behavior
+-------------------------------
+
+<!--
+The more time you spend describing an easy way to reproduce the behavior (if
+this is possible), the easier it is for the project developers to fix it!
+-->
+
+
+Do you have any idea what may have caused this?
+-----------------------------------------------
+
+
+
+Do you have an idea how to solve the issue?
+-------------------------------------------
+
+
+
+Did XCSoar help you or make you happy in any way?
```

Comment:
I'm not interested at all in this. That's just fishing for compliments.

Reply:
Understood. But what about the other core developers? Is *noone* interested in hearing a bit of positive feedback from users while they're sitting here doing voluntary work?

Reply:
Maybe, but this is the wrong place for compliments.

Reply:
I removed it entirely.